ocrd/ocrd/workspace.py

```diff
@@ -194,3 +205,184 @@ def resolve_image_as_pil(self, image_url, coords=None):
             np.min(poly[:, 0]):np.max(poly[:, 0])
         ]
         return Image.fromarray(region_cut)
+
+    def image_from_page(self, page, page_id):
+        """Extract the Page image from the workspace.
+
+        Given a PageType object, `page`, extract its PIL.Image from
+        AlternativeImage if it exists. Otherwise extract the PIL.Image
+        from imageFilename and crop it if a Border exists. Otherwise
```

Comment:
@wrznr is right about insisting that the term `cropping` (de: `Beschneidung`) should only apply to the process of finding the `Border` (and perhaps also removing the margins from the image by cutting), not of other elements down the hierarchy. This we should rather call `cutting` (de: `Freistellen`) – it is only due to `PIL.Image.crop` that I was led astray.

If this is correct, then the docstrings must be fixed accordingly throughout.

Reply:
Not a huge fan because I find the distinction unintuitive but I see the reasoning. Should `crop_image` then be renamed `cut_image` as well since image isn't necessarily a page?

Reply:
That would be more consequential, but I tend to say no: `crop_image` is meant as replacement for `Image.crop` and should be memorable. I am becoming less enthusiastic about this terminological distinction by the minute... maybe this should be reverted (sorry).

Reply:
cf #289